include/exadg/incompressible_navier_stokes/user_interface/enum_types.h

```diff
@@ -370,6 +370,7 @@ enum class PreconditionerPressurePoisson
 {
   None,
   PointJacobi,
+  BlockJacobi,
```

Comment:
was this just forgottten?

Reply:
no, I added this preconditioner variant here. You are right in the sense that this type of preconditioner would have been available before the present PR. The reason is probably historically, i.e. the block-Jacobi preconditioner was added later to ExaDG, while for the pressure Poisson equation, the geometric multigrid preconditioner is our working horse.